lib/travis/build/script/d.rb

```diff
@@ -18,22 +19,34 @@ def setup
           super
 
           sh.echo 'D support for Travis-CI is community maintained.', ansi: :green
-          sh.echo 'Please make sure to ping @MartinNowak, @klickverbot and @ibuclaw '\
+          sh.echo 'Please make sure to ping @MartinNowak and @wilzbach'\
             'when filing issues under https://github.com/travis-ci/travis-ci/issues.', ansi: :green
 
+          sh.echo 'DMD-related issues: https://issues.dlang.org', ansi: :green
+          sh.echo 'LDC-related issues: https://github.com/ldc-developers/ldc/issues', ansi: :green
+          sh.echo 'GDC-related issues: https://bugzilla.gdcproject.org', ansi: :green
+          sh.echo 'DUB-related issues: https://github.com/dlang/dub/issues', ansi: :green
+
           sh.fold 'compiler-download' do
             sh.echo 'Installing compiler and dub', ansi: :yellow
 
             sh.cmd 'CURL_USER_AGENT="Travis-CI $(curl --version | head -n 1)"'
             sh.cmd <<-'EOT'.gsub(/^              /, '')
+              __mirrors=(
+                "https://dlang.org/install.sh"
+                "https://downloads.dlang.org/other/install.sh"
+                "https://nightlies.dlang.org/install.sh"
+                "https://raw.githubusercontent.com/dlang/installer/0ddce7a399eda9201db3eab90c6ad073d92ac1c2/script/install.sh"
```

Comment:
Is this the best URL? Would `https://github.com/dlang/installer/raw/0ddce7a399eda9201db3eab90c6ad073d92ac1c2/script/install.sh` be more stable?

Reply:
I don't think that `raw.githubusercontent.com` is going to vanish soon.
The direct GitHub link would introduce another point of failure:

```
> curl -v https://github.com/dlang/installer/raw/0ddce7a399eda9201db3eab90c6ad073d92ac1c2/script/install.sh
<html><body>You are being <a href="https://raw.githubusercontent.com/dlang/installer/0ddce7a399eda9201db3eab90c6ad073d92ac1c2/script/install.sh">redirected</a>.</body></html>
```

Reply:
Well, you would need to follow redirects using `-L`.

Reply:
I'm pretty sure I've seen invalid URLs in the wild which point to raw GitHub content that have a different domain name.

Reply:
Fair enough. Done.